app/services/report/docx_action/experiment.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# rubocop:disable  Style/ClassAndModuleChildren
+module Report::DocxAction::Experiment
```

Comment:
Do we have any good practice about how to name services? I'd say it's probably better to use e.g. `app/services/reports/generate_docx/...` or something like that.